src/ape/managers/networks.py

```diff
@@ -97,10 +98,7 @@ def ecosystems(self) -> Dict[str, EcosystemAPI]:
             ecosystem_config = self.config_manager.get_config(plugin_name).dict()
             default_network = ecosystem_config.get("default_network", LOCAL_NETWORK_NAME)
 
-            if default_network and default_network in ecosystem.networks:
```

Comment:
I removed these parts because we already raise custom errors in the `set_default_()` methods